doc/doxyfile.in

```diff
@@ -44,14 +44,14 @@ PROJECT_NUMBER         = @VERSION_MAJOR@.@VERSION_MINOR@.@VERSION_PATCH@
 # for a project that appears at the top of each page and should give viewer a
 # quick idea about the purpose of the project. Keep the description short.
 
-PROJECT_BRIEF          = 
+PROJECT_BRIEF          =
```

Comment:
If it is generated, then by some helper program, but it isn't modified by the primary doxygen executable itself. At the end of the day, we are the ones who are supposed to manually edit the file to customize doxygen anyway. Even when updating Doxygen, we wouldn't "automatically regenerate" this file, since that means we would have to tweak the generated file with our custom settings again as well.